app/src/main/assets/GimelGimel/AndroidAPI.js

```diff
@@ -11,11 +11,21 @@ GG.AndroidAPI = {
      * @param location - location to push into Android for update.
      * Should be an object with latitude, longitude properties
      */
-    updateLocation: function (location) {
+    updateSelectedLocation: function (location) {
         var locationJsonString = JSON.stringify(location);
         LocationUpdater.UpdateSelectedLocation(locationJsonString);
     },
     /**
+     * Updated android via injected JavascriptInterface
+     *
+     * @param location - location to push into android for update.
+     * Should be an object with latitude, longitude and
```

Comment:
"Should be an object with latitude, longitude and"
and what ? such a cliffhanger.... !resolve